test/event.test.js

```diff
@@ -1,15 +1,15 @@
 'use strict'
 const helper = require('./helper')
 const ao = helper.ao
-const addon = ao.addon
+//const aob = ao.addon
```

Comment:
gone?

Reply:
used to be a requirement when it needed the c++ functions. removed.